### ISSUE TITLE
"Platform Agnostic" on "What is a Component" isn't clear

### ISSUE THREAD

**Issue by jacksonj04** [poplus/home-poplus#32]
On http://poplus.org/what-is-a-component.html a Poplus component criteria is described as "platform agnostic", but it doesn't appear to mean platform agnostic in the commonly accepted sense (ie the software can run on any platform). In fact, the phrase "Write your Component in whatever language you are most comfortable with" seems to explicitly make this not the case (eg I may be most comfortable in C#, which would realistically limit me to a Windows environment).

Instead, what it seems to say is that components should be _consumable_ by other software on any platform, ie they must use open, cross-platform standards in their interfaces.

**Comment by maugsbur** [comment]
You're right! Does it sound ok to add "consumable by other software on any
platform" in parenthesis?

2014-05-20 6:28 GMT-04:00 Nick Jackson notifications@github.com:
